examples/04_glm_first_level/plot_write_events_file.py

```diff
@@ -71,13 +69,21 @@
 
 trial_types = [condition_ids[i] for i in trial_type_idx]
 
-#########################################################################
+# %%
 # We must also define a duration (required by :term:`BIDS` conventions).
-# In this case, all trials lasted one second.
-
-durations = np.ones_like(onsets)
-
-#########################################################################
+# In this case, all trials lasted one second,
+# except for button response
+# that are modeled as events with instantaneous duration (0 second).
+
+null_duration_trials = [2, 3, 4, 5]
+durations = []
+for i in trial_type_idx:
+    if i in null_duration_trials:
+        durations.append(0)
+    else:
+        durations.append(1)
```

Comment:
@bthirion 
I took the liberty to modify this example to show the 0 duration.
Is that OK or would you prefer to stay true to what was done in the paper?

Reply:
No pb.